src/SignalR/server/Specification.Tests/src/HubLifetimeManagerTestBase.cs

```diff
@@ -8,12 +8,23 @@
 
 namespace Microsoft.AspNetCore.SignalR.Specification.Tests
 {
+    /// <summary>
+    /// Base test class for lifetime manager implementations. Nothing specific to scale-out for these tests.
+    /// </summary>
+    /// <typeparam name="THub">The type of the <see cref="Hub"/>.</typeparam>
     public abstract class HubLifetimeManagerTestsBase<THub> where THub : Hub
     {
-        public HubLifetimeManager<THub> Manager { get; set; }
-
+        /// <summary>
+        /// Method to create an implementation of <see cref="HubLifetimeManager{THub}"/> for use in tests.
+        /// </summary>
+        /// <returns>The implementation of <see cref="HubLifetimeManager{THub}"/> to test against.</returns>
         public abstract HubLifetimeManager<THub> CreateNewHubLifetimeManager();
 
+        /// <summary>
+        /// Creates a couple connections, registers them with the lifetime manager, then checks that
```

Comment:
~~couple~ --> few or _couple of_.

I'd also consider just describing this as

> Specification test for SignalR HubLifetimeManager. 

Seems overkill to describe what the test does.